doc/source/development/rolling_operations_with_numba.rst

```diff
@@ -0,0 +1,173 @@
+Rolling Operations with Numba
+=============================
+
+Summary
+-------
+
```

Comment:
let’s add a Future section
- use jitclasses (mention issues and resolution: https://github.com/numba/numba/issues/4522#issuecomment-537872456)

can include what we want short term (implement rest of the aggregations, expanding, ewma)

and longer term; groupby & groupby + rolling